CSS-Example-Style-Guide.md

```diff
@@ -0,0 +1,64 @@
+
+To avoid vertical scrollbars, you have space for 6 one-line choices. If the property can take more than 6 forms (for example, for all the different [`filter`](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) functions) then it is best to choose 6 that are illustrative. It's *not* essential to have an example for every form.
+
+If the property takes fewer than 6 forms, it's fine to show fewer. For example: [`text-overflow`](https://interactive-examples.mdn.mozilla.net/pages/css/text-overflow.html).
+
+### Horizontal overflow
+
+If any CSS declaration overflows its container horizontally, then the container for its choice will get a horizontal scrollbar, and the part of the declaration that overflows will not be visible without scrolling. This isn't as bad for discoverability as vertical overflow, but we'd still prefer to avoid it.
+
+So try to be concise in the syntax you use, but not at the expense of making the code understandable and good practice. Quite often you won't be able to avoid horizontal scrollbars: see [`background-image`](https://interactive-examples.mdn.mozilla.net/pages/css/background-image.html) for example.
+
+### Multiple declarations
+
+Sometimes you'll want to show extra CSS declarations, in addition to the one for the property you are illustrating. This is slightly unusual, though, and should be used only where the extra properties are closely related to the property being illustrated.
+
+For example, in the example for [`position`](https://interactive-examples.mdn.mozilla.net/pages/css/position.html) we show `top` and `left` for forms like `position: relative;`, because they're needed to understand the effect of these declarations, and because we want to invite people to play around with those values.
+
+But be restrained with this: including unrelated declarations makes it harder to see what a specific example is intended to illustrate.
+
+If you have multiple declarations, each one should occupy its own line. Note that [`position`](https://interactive-examples.mdn.mozilla.net/pages/css/position.html) breaks this rule, because otherwise it would be impossible to fit relevant forms without having a vertical scrollbar. This is quite unsatisfactory though, and is only acceptable here because `top` and `left` are closely related to each other, and are only there in a "supporting" role: they are not the property being illustrated.
+
+### Comments
+
+Since vertical space is at a premium, avoid comments unless they are really needed. Keep in mind that these are examples, not documentation in themselves.
+
+### Initial choices
+
+By default, the first choice will be selected. You can override this by adding `initial-choice="true"` to the `example-choice` DIV:
```

Comment:
Note to self: Should probably change this to be a data attribute for consistency.